Aqovia.Utilities.SagaMachine/StatePersistance/RedisKeyValueStore.cs

```diff
@@ -106,6 +106,54 @@ public void Remove(string key)
             db.KeyDelete(key);
         }
 
+
+        /// <summary>
+        /// Request lock against key value element
+        /// Lock lifespan 500 milliseconds
+        /// </summary>
+        /// <param name="key"></param>
+        /// <param name="lockToken"></param>
+        /// <returns></returns>
+        public bool TakeLockWithShortTimeSpan(string key, out string lockToken)
+        {
+            lockToken = Guid.NewGuid().ToString();
+
+            var db = GetDatabase();
+            var transac = db.CreateTransaction();
+            transac.AddCondition(Condition.KeyExists(key));
+
+            return transac.LockTakeAsync(key, lockToken, TimeSpan.FromMilliseconds(500)).Result;
```

Comment:
Can you just change it to reuse the other member? Like:
`TakeLock(key, out lockToken, milliseconds: 500)`